source/common/http/http2/codec_impl.h

```diff
@@ -169,8 +169,9 @@ class ConnectionImpl : public virtual Connection,
     }
   }
   void onUnderlyingConnectionBelowWriteBufferLowWatermark() override {
-    for (auto& stream : active_streams_) {
-      stream->runLowWatermarkCallbacks();
+    // Notify the streams based on least recently encoding to the connection.
+    for (auto it = active_streams_.rbegin(); it != active_streams_.rend(); it++) {
+      (*it)->runLowWatermarkCallbacks();
```

Comment:
I notice that the code in  `updateActiveStreamsOnEncode()` is guarded by `stream.defer_processing_backedup_streams_` but the change here to process the list in reverse order is not. Can you say more about this?